src/shared/request-context/request-context.dto.ts

```diff
@@ -2,11 +2,8 @@ import { UserAccessTokenClaims } from '../../auth/dtos/auth-token-output.dto';
 
 export class RequestContext {
   public requestID: string;
-
   public url: string;
-
   public ip: string;
-
   // TODO : Discuss with team if this import is acceptable or if we should move UserAccessTokenClaims to shared.
```

Comment:
In my opinion, main modules should not be imported into shared modules.
Rather than using the UserAccessTokenClaims type, can we just mention an interface in shared for User?

Reply:
created a separate issue for this one for future enhancement. 

https://github.com/monstar-lab-oss/nestjs-starter-graphql-api/issues/3